### ISSUE TITLE
Requesting to make a video tutorial for integrating voice in Rasa

### ISSUE THREAD

**Issue by mhmohona** [RasaHQ/rasa#5235]
I was following this [tutorial](https://blog.rasa.com/how-to-build-a-voice-assistant-with-open-source-rasa-and-mozilla-tools/) and found in some places it is hard for me to understand what has been told to do. It is not only me, in Stackoverflow and forum I found many are struggling about this topic too. 
It would be great if there were a tutorial particularly focusing on integrating voice using the Mozilla Deepspeech tool.

**Comment by sara-tagger** [comment]
Thanks for the issue, @dakshvar22 will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by dakshvar22** [comment]
@mhmohona Could you explain what parts of the tutorial were hard to follow?

**Comment by TyDunn** [comment]
@mhmohona Let us know on the Rasa forum if you are still having trouble integrating Rasa with voice :)
